app/src/main/java/org/fossasia/phimpme/base/BaseActivity.java

```diff
@@ -86,6 +86,7 @@ public void onShow(MaterialShowcaseView itemView, int position) {
                         .setTarget(nav_cam)
                         .setDismissText(getResources().getString(R.string.ok_button))
                         .setContentText(getResources().getString(R.string.camera_button))
+                        .hideOnTouchOutside()
```

Comment:
Try using using `.setDismissOnTouch(true)` instead

Reply:
Did it and worked.